FreeRTOS/FreeRTOS
# Fix issue in CORTEX_LM3S811_GCC Demo linkerscipt

The .text section is listed as *(.text) and results in build
error because there are some .text* sections that are automatically put
into flash later by linker and their LMA overlaps
with .data



Description
-----------

explained in description
Test Steps
-----------

To reproduce, go to the CORTEX_LM3S811_GCC demo  folder and build with gcc-arm-none-eabi.

Related Issue
-----------

n/a

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
